client/app/hosting/framework/envvar/FRAMEWORK_ENVVARS.html

```diff
@@ -0,0 +1,126 @@
+        </button>
+
+        <div class="container-fluid px-0">
+            <h6 data-i18n-static="hosting_tab_FRAMEWORK_envvar_list"></h6>
+            <div class="mb-3">
+                <form class="form-inline d-md-flex justify-content-md-end" name="searchEnvvarForm">
+                    <div class="form-group">
+                        <label class="sr-only" for="searchEnvvar" data-ng-i18n-static="common_search"></label>
+                        <div class="input-group">
+                            <input type="text" class="form-control" id="searchEnvvar" name="searchEnvvar" placeholder="{{::i18n.hosting_tab_FRAMEWORK_envvar_search_placeholder}}"
+                                   data-ng-model="ctrlEnvvar.search"
+                                   data-ng-change="ctrlEnvvar.onSearch(ctrlEnvvar.search)"
+                                   data-ng-model-options='{ debounce: 800 }'>
+
+                            <div class="input-group-btn" data-ng-if="ctrlEnvvar.search">
+                                <button class="btn btn-default" type="button" aria-label="{{i18n.common_cancel}}"
+                                        data-ng-click="ctrlEnvvar = ''">
+                                    <span class="fa fa-times" aria-hidden="true"></span>
+                                </button>
+                            </div>
+
+                            <span class="input-group-addon" data-ng-if="!ctrlEnvvar.search">
+                        <span class="fa fa-search" aria-hidden="true"></span>
+                    </span>
+                        </div>
+                    </div>
+                </form>
+            </div>
+
+            <div class="table-responsive">
+                <table class="table table-hover">
+                    <thead>
+                    <tr>
+                        <th scope="col" data-i18n-static="hosting_tab_FRAMEWORK_envvar_table_header_key"></th>
+                        <th scope="col" data-i18n-static="hosting_tab_FRAMEWORK_envvar_table_header_type"></th>
+                        <th scope="col" data-i18n-static="hosting_tab_FRAMEWORK_envvar_table_header_value"></th>
+                        <th scope="col" data-i18n-static="hosting_tab_FRAMEWORK_envvar_table_header_status"></th>
+                        <th class="min-width text-right" scope="col">
+                            <button class="btn btn-icon" type="button" title="{{i18n.refresh}}"
+                                    data-ng-click="onRefresh()"
+                                    data-ng-disabled="ctrlEnvvar.loading.envvars">
+                                <span class="fa fa-refresh" aria-hidden="true"></span>
+                            </button>
+                        </th>
+                    </tr>
+                    </thead>
+
+                    <tbody data-ng-if="ctrlEnvvar.loading.envvars">
+                    <tr>
+                        <td class="loader text-center" colspan="5"></td>
```

Comment:
You have to use `<oui-spinner>` component.

```html
<td class="loader text-center" colspan="5"></td>
```

to

```hmtl
<td colspan="5"
    class="text-center">
    <oui-spinner data-size="s"></oui-spinner>
</td>
```

Bonus: You can use `<oui-datagrid>` if you feel confortable with (ref: https://master.ui-kit.ovh/#!/oui-angular/datagrid)